examples/showcase/tracked-controls/js/components/collider.js

```diff
@@ -0,0 +1,109 @@
+/* global AFRAME, THREE */
+
+/**
+ * It implements AABB collision detection for entities with a mesh.
+ * (https://en.wikipedia.org/wiki/Minimum_bounding_box#Axis-aligned_minimum_bounding_box)
+ * It sets the specified state on the intersected entities.
+ *
+ * @property {string} Selector of the entities to test for collision.
+ * @property {string} State to set on collided entities.
+ *
+ */
+AFRAME.registerComponent('collider', {
+  schema: {
+    objects: {default: ''},
+    state: {default: 'collided'}
+  },
+
+  init: function () {
+    this.els = [];
```

Comment:
The raycaster sets `this.intersectedEls`. Could be consistent with `this.intersectedEls` or `this.collidedEls`.